ek9000App/src/devEK9000.cpp

```diff
@@ -204,8 +204,8 @@ devEK9000Terminal* devEK9000Terminal::ProcessRecordName(const char* recname, int
 		for (size_t i = len; i >= 0; i--) {
 			if (ret[i] == ':' && (size_t)i < len) {
 				ret[i] = '\0';
-				good = true;
-				*outindex = atoi(&ret[i + 1]);
+				if (epicsParseInt32(ret + 1, outindex, 10, NULL) == 0)
```

Comment:
Was this where your `parseInt` utility should have come in?

Reply:
Yep, `epicsParseInt32` works fine though. The return value is a bit weird so I was trying to avoid the footgun with `parseInt`.